hashintel/hash
# Improve alignment of page title and layout

## 🌟 What is the purpose of this PR?

Improve the page title and page layout styling to match the design.

## 🔗 Related links

- [Asana task](https://app.asana.com/0/1200211978612931/1202782394094087/f) _(internal)_
- [Design](https://www.figma.com/file/gydVGka9FjNEg9E2STwhi2/HASH-App?node-id=1503%3A124892) _(internal)_


## 🔍 What does this change?

- Used a `h1` with `contenteditable=true` as `PageTitle` instead of `input`. So now it's SEO friendly & accessible 

## ⚠️ Known issues
 https://github.com/hashintel/hash/blob/1997d06ccfda38f9912c73d7af66e3a792e25d85/packages/hash/frontend/src/blocks/page/focusPageTitlePlugin.ts#L14-L22

## ❓ How to test this?



1.  Checkout the branch / view the deployment
1.  Open a page
2. Check the title behavior, pressing up-arrow from the first block on the page should focus to the page title
3. Compare the Figma design with website. They should look identical

## 📹 Demo

### Comparison of old and new page layouts

Old Layout             |  New Layout
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/39553853/185431216-c1f17728-04c7-4b7e-8aa3-3eca4f3e9efe.png)  |  ![](https://user-images.githubusercontent.com/39553853/185431207-8a6e1893-5118-440a-8471-430f359e39a5.png)
